### PR TITLE
jshint and flake8 cleanup

### PR DESCRIPTION
**Pull request by brutal-chaos**
Code cleanup, both python and javascript.

### CODE REVIEW

File: orvsd_central/util.py [hunk 1]
```diff
@@ -311,7 +311,7 @@ def gather_tokens(sites=[], service_names=[]):
 
         # For the request, prepend the protocol if necessary
         site_url = ("http://%s" % site.baseurl
-            if not site.baseurl.startswith("http") else site.baseurl)
+                    if not site.baseurl.startswith("http") else site.baseurl)
```
Comment (Kennric) [inline orvsd_central/util.py]: Why the extra-wide indent?

**Comment by Kennric** [comment]
This looks pretty good, have you tested the javascript to make sure no errors were introduced?

**Comment by brutal-chaos** [comment]
I believe so. I tested most if not all pages, added a site, installed a course. All worked well.

File: orvsd_central/util.py (hunk 1 above)
Comment (brutal-chaos) [inline orvsd_central/util.py]: Pep8 says it was under indented. to align, it needed to be under the ".

Comment (Kennric) [inline orvsd_central/util.py]: Oh, I see it now, it looks odd to have an if block inside the (

**Comment by Kennric** [comment]
Looks good to me +1 to merge

**Pull request merged**